apps/docs-snippets/test/fixtures/forc-projects/bytecode-input/Forc.toml

```diff
@@ -0,0 +1,7 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+license = "Apache-2.0"
+name = "bytecode-input"
+
+[dependencies]
+bytecode = { git = "https://github.com/FuelLabs/sway-libs", tag = "v0.19.0" }
```

Comment:
Why do we need to pin for an older version? My concern is that we tend to keep all our examples fixed to a specific verison.

Reply:
Absolutely, good catch! I second this concern. ☝️